go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -179,14 +183,28 @@ func NewThrottler(env tabletenv.Env, ts *topo.Server, heartbeatWriter heartbeat.
 
 		throttler.httpClient = base.SetupHTTPClient(2 * mysqlCollectInterval)
 		throttler.initThrottleTabletTypes()
-		throttler.ThrottleApp("abusing-app", time.Now().Add(time.Hour*24*365*10), defaultThrottleRatio)
+		throttler.ThrottleApp("always-throttled-app", time.Now().Add(time.Hour*24*365*10), defaultThrottleRatio)
 		throttler.check = NewThrottlerCheck(throttler)
 		throttler.initConfig()
 		throttler.check.SelfChecks(context.Background())
+	} else {
+		// Create an empty cache, just so that it isn't nil
+		throttler.throttledApps = cache.New(cache.NoExpiration, 0)
 	}
 	return throttler
 }
 
+// CheckIsReady checks if this throttler is ready to serve. If not, it returns an error
+func (throttler *Throttler) CheckIsReady() error {
+	if !throttler.isEnabled {
```

Comment:
super nit-picking here...
instead of two `if`s you could have a single one with an OR 
and
`ErrThrottlerNotready` -> `ErrThrottlerNotReady`

Reply:
Fixed typo and fixed `if`!